src/dbnode/client/options.go

```diff
@@ -327,26 +328,6 @@ func NewOptionsForAsyncClusters(opts Options, topoInits []topology.Initializer,
 	return result
 }
 
-func defaultNewConnectionFn(
```

Comment:
moved to connection_pool.go and split into 2 methods: NewConnectionFn and NewClientFn